app/models/form/question.rb

```diff
@@ -53,10 +53,7 @@ def answer_label(log)
     answer = label_from_value(log[id]) if log[id].present?
     answer_label = [prefix, format_value(answer), suffix_label(log)].join("") if answer
 
-    inferred = inferred_check_answers_value["value"] if inferred_check_answers_value && has_inferred_check_answers_value?(log)
-    return inferred if inferred.present?
-
-    answer_label
+    inferred_answer_value(log) || answer_label
```

Comment:
Does this not return a true/false value as it's a boolean evaluation? If so, how does that work given previously this method output an formatted answer to a question?

Reply:
Fair enough, thanks!

Reply:
I can't find this documented anywhere. Could you point me to some other examples in the codebase please? :) (Or actual documentation if you can find any. Thanks!)

Reply:
Hmm, maybe this https://www.visuality.pl/posts/what-does-the-operator-actually-mean-in-ruby `The || operator` part.
Some examples in the codebase:
https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/app/controllers/schemes_controller.rb#L262
https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/app/components/log_summary_component.html.erb#L62
https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/app/controllers/users_controller.rb#L49

Let me know if you wanted more, but they're all similar

Reply:
Amazing, thanks! Good to see it in action. Wish I could find it in the official docs for Ruby, but oh well 😁